data/mods/MindOverMatter/powers/teleportation.json

```diff
@@ -101,6 +101,42 @@
     "final_casting_time": 15,
     "casting_time_increment": -3
   },
+  {
+    "id": "teleport_phasing",
+    "type": "SPELL",
+    "name": "[Ψ]Phasing (C)",
+    "description": "By moving slightly into the void between dimensions, you can exist in a state that allows effortless travel through obstacles.",
+    "message": "You feel a strange coolness as matter around you feels somehow less solid.",
```

Comment:
This message should be moved to the EoC, otherwise it'll trigger when you toggle the power off too.